### DISCUSSION TITLE
Lighthouse Feature - Options to Measure User Flow Performance in CL

### DISCUSSION THREAD

**Discussion by csharsha** [GoogleChrome/lighthouse#14277] [Ideas]
**Feature request summary**

Is there an option to measure User flow performance like running timespan API or taking snapshots through command line? If yes how to use it. If not, is there any plan to bring this feature in near future?

**What is the motivation or use case for changing this?**
This would help to easily integrate Lighthouse with Selenium and other automation tools.

**Comment by adamraine** [comment]
There is currently no user flow CLI, although we are considering ways to do this using DevTools recordings related to #14146. However, we don't have any plans to support Selenium this way.

You could always create own CLI entry point using the user flow API in a JS script.
